stylesheets/design-patterns/_media-player.scss

```diff
@@ -222,7 +227,9 @@
         height: 100%;
         padding-bottom: 1px;
         width: 20px;
-        color: rgba(0,0,0,0.5);
+        // scss-lint:disable ColorVariable
+        color: rgba(0, 0, 0, 0.5);
+        // scss-lint:enable ColorVariable
```

Comment:
I'm not sure this needs to be re-enabled, when disabled within a block I _believe_ the linter is disabled for just that block: https://github.com/brigade/scss-lint (see "Disable all lints within a block (and all contained blocks)")